assets/i18n/en.json

```diff
@@ -22,7 +22,7 @@
   "START_NOW": "Start now",
   "IF_YOU_HAVE_ANY_QUESTIONS_REG_CHB": "If you have any questions about Child Benefit ",
   "CONTACT_THE_CHILD_BENEFIT_HELPLINE": "contact the Child Benefit helpline",
-  "PAGE_NOT_WORKING_PROPERLY": "Is this page not working properly? (opens in new tab)",
+  "PAGE_NOT_WORKING_PROPERLY": "Is this page not working properly?",
```

Comment:
this change is going to affect the footer as well, please make sure you add the text (opens in a new tab) in footer as well

Reply:
Also the corresponding change should be done in cy.json

Reply:
resolved